low-level/framework/utils/filestore.py

```diff
@@ -129,9 +129,17 @@ def _load_json_file(self, key):
         return value
 
     def exists(self, key):
-        """check if key exists
+        """check if key is present
         """
-        return os.path.exists(key)
+        key_present = False
+        status = "Failure"
+        try:
+            key_present = os.path.exists(key)
+            status = "Success"
+        except Exception as gerr:
+            logger.warn("Error{0} while checking if key is present".format(gerr))
```

Comment:
logger.warn("Error while checking if {0} is present".format(gerr))

Reply:
done